content/user/admin/why_admin.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: What's the purpose of the admin? 
+description: Why so big?  Also, where's the feature *I* want? 
```

Comment:
I wonder whether "big" is the best way to describe this?

Reply:
updated to say "so many options."